discovery-provider/src/tasks/index_rewards_manager.py

```diff
@@ -453,6 +516,7 @@ def process_transaction_signatures(
     last_tx: Optional[RewardManagerTransactionInfo] = None
     if transaction_signatures and transaction_signatures[-1]:
         last_tx_sig = transaction_signatures[-1][0]
+    challenge_type_map: Dict[str, TransactionType] = {}
```

Comment:
would probably be simpler to just have this challenge_type_map be a variable at the top level scope of this file, and have get_challenge_type_map either return that or write & return - avoiding passing around the map through multiple fns